### PR TITLE
Add Psycho Theme to V2

### PR DESCRIPTION
**Pull request by harbassan**

### CODE REVIEW

**Comment by harbassan** [comment]
Well it is similar, but the UI has been slightly changed such as the player, which has the playback bar spanning the full width of the window, similar to the legacy SpicetifyDefault theme, along with the subheadings, and a few changes to the variables used to colour certain objects. That's about it really.

**Comment by morpheusthewhite** [comment]
I think you could push it as color scheme to the existing `Default` theme since it is very similar.

Also I think I would be better to increase the contrast between the red and black to improve readability.

**Comment by harbassan** [comment]
I mean considering that of the original legacy themes, a large amount have the playback bar as normal in the player, and a large amount have it as I do above the player, wouldn't it make sense to separate these into two different themes based on that? In the migration process you have.

**Comment by morpheusthewhite** [comment]
If you ignore the size of the player then the only visible difference is the play button if I'm correct, so I think it's not necessary to create a different theme for that

**Comment by harbassan** [comment]
I don't think you can tell from the screenshot I provided, maybe you can but I'm not sure.
Screenshot
Screenshot 2021-05-21 191454
Many themes use either of these two, so it would be sensible to make two different themes because of this, right? That's what I saw in the themes you categorized in the migration process right? Sorry if you understand this already I'm just making sure.

**Comment by morpheusthewhite** [comment]
You're, I overlooked the screenshot you uploaded, thanks for letting me know. Just update the colours and fix the conflicts and I'll merge it. 

I would also appreciate a more neutral theme name since we may want to push the color schemes I listed in the PR you mentioned, but I can't come up with a better one.

**Comment by harbassan** [comment]
I've made the changes to the theme now. That was the best name I could think of.

File: THEMES.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ Here you can find a preview of all the themes. Some of them may have different c
...
+![Psycho Screenshot](Sleek/psycho.png)
+
+#### Deep
+
+![Psycho Screenshot](Sleek/deep.png)
+
 ## Onepunch
 
-![Onepunch](Onepunch/\__Home.png)
+![Onepunch Screenshot](Onepunch/\__Home.png)
```
Comment (morpheusthewhite) [inline THEMES.md:21]: Put the themes in alphabetical order

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]

**Comment by harbassan** [comment]
done

**Review by morpheusthewhite — APPROVED** [review]

**Pull request merged**